package.json

```diff
@@ -146,6 +146,7 @@
     "ava": "^4.3.1",
     "better-sqlite3": "^7.6.2",
     "eslint": "^8.22.0",
+    "expo-sqlite": "^11.0.0",
```

Comment:
All looks great. Just one question is whether there's any way to avoid this as a straight dev dependency. We can rely on it to be installed as an optional peer dependency when actually in use.

Reply:
Why do you want to avoid that? It won't be pulled in as a transitive dependency anyway. I only use it to import types.

Reply:
To avoid installing in development for users not using expo.

Reply:
It won't, if you're talking about `npm install electric-sql`, that never pulls in dev dependencies. Otherwise it would pull in prettier, eslint, and the rest

Reply:
Right, sorry, yes, I get it. It's only a problem if you're actually developing the package.